style.css

```diff
@@ -0,0 +1,77 @@
+/*   Small phones from 0 to 480px  */
+
+@media only screen and (max-width: 500px) {
```

Comment:
- [ ] There seems to be no styling when the app is adjusted past 500px because of the media query. Please fix it.

- [ ] **[OPTIONAL]** Since your app is expected to be **mobile-first** anyway, I don't think a media query is required. You can just build your features with a mobile view in mind. You can use the developer tools for inspecting what your app looks like on mobile.